services/worker-manager/src/data.js

```diff
@@ -7,18 +7,33 @@ const WorkerType = Entity.configure({
   version: 1,
   partitionKey: Entity.keys.StringKey('workerTypeName'),
   rowKey: Entity.keys.ConstantKey('workerType'),
+  properties: {
+    workerTypeName: Entity.types.String,
+    provider: Entity.types.String,
+    previousProviders: Entity.types.JSON,
+    description: Entity.types.String,
+    scheduledForDeletion: Entity.types.Boolean,
+    created: Entity.types.Date,
+    lastModified: Entity.types.Date,
+    config: Entity.types.JSON,
+    owner: Entity.types.String,
+    emailOnError: Entity.types.Boolean,
+    providerData: Entity.types.JSON,
+  },
+}).configure({
```

Comment:
Honestly I'd prefer we just break things again to keep this file simpler. I doubt many people have deployed a cluster in between the last breaking change and this one.

Reply:
Fair enough